projects/media-viewer/src/lib/store/actions/document.action.ts

```diff
@@ -10,6 +11,12 @@ export const CONVERT_SUCCESS = '[Document] Convert Success';
 export const CONVERT_FAIL = '[Document] Convert Fail';
 export const CLEAR_CONVERT_DOC_URL = '[Document] Clear Convert Doc Url';
 
+export const LOAD_ROTATION = '[Document Load Rotation]'
```

Comment:
Should it be in the format '[Document] Load Rotation' to match the other strings?